src/flutter/shell/platform/linux_embedded/surface/environment_egl.h

```diff
@@ -12,10 +12,9 @@
 
 namespace flutter {
 
-template <typename T>
 class EnvironmentEgl {
```

Comment:
Please create environment_egl.cc file.

Reply:
I didn't create environment_egl.cc file because the include order of `gbm.h` causes a link error. I added this description to environment_egl.h.

Reply:
```
The definition of EGLNativeDisplayType depends on related include files or define such as gbm.h or "__GBM__". So, we cannot create environment_egl.cc file and need to avoid a link error which is caused by the include order of related header files.
```

Reply:
I fixed it.